documentation/applying_manifest_blocks.md

```diff
@@ -16,7 +16,7 @@ Similar to the `puppet apply` command, which applies a standalone Puppet manifes
 
 ## How manifest blocks are applied
 
-When you run a plan that contains a manifest block, the `apply_prep` function installs the packages necessary to run the Bolt `apply` command.
+When you run a plan that contains a manifest block, 99% of the time you want to include the `apply_prep` function in your plan before your manifest block.  `apply_prep` installs the packages necessary to run the Bolt `apply` command and gathers facts by running `facter` making the facts available to manifest blocks.  Without any facts manifest blocks won't compile.  
```

Comment:
```suggestion
When you run a plan that contains a manifest block, 99% of the time you want to include the `apply_prep` function in your plan before your manifest block. `apply_prep` ensures the necessary packages to run the Bolt `apply` are installed and gathers facts by running `facter` making the facts available to manifest blocks.  Without common facts most manifest blocks won't compile.  
```

Reply:
I don't like 99% but I'm not sure whats better.

Reply:
The alternate wording could be more around what the mechanism is.  "when you run a plan that contains a manifest block, the targets the manifest block is running against need to have facts set on them.  The easiest way to ensure facts are set is to run `apply_prep` on those targets before the manifest block.  ....

Reply:
I'm going to push up a change to this quick.

Reply:
I'm going to push up a change to this